xform-marc21-to-xml/src/main/java/edu/stanford/AuthDBProperties.java

```diff
@@ -76,6 +80,37 @@ private void initDataSourceProperties(Properties properties) {
         this.userPass = properties.getProperty("PASS");
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
It seems to me that these methods exist only for the sake of the `AuthDBPropertiesTest.class`, true? So if we can use some library's code for this purpose instead of having some cryptic-looking methods in the functional class, I would be for that change.